cmd/lakefs/cmd/import.go

```diff
@@ -44,7 +46,9 @@ var importCmd = &cobra.Command{
 		dryRun, _ := flags.GetBool(DryRunFlagName)
 		manifestURL, _ := flags.GetString(ManifestURLFlagName)
 		withMerge, _ := flags.GetBool(WithMergeFlagName)
-		hideProgress, _ := flags.GetBool(HideProgress)
+		hideProgress, _ := flags.GetBool(HideProgressFlagName)
+		prefixFile, _ := flags.GetString(PrefixesFileFlagName)
```

Comment:
I think maybe passing in multiple `--prefix` flags would be easier? It's possible using [pflag.StringArray](https://pkg.go.dev/github.com/spf13/pflag#StringArray)

Reply:
It's possible, just that it can potentially become a huge flag, in an already bloated command